tools/export/exporters.py

```diff
@@ -130,9 +131,13 @@ def make_key(self, src):
         Positional Arguments:
         src - the src's location
         """
-        key = basename(dirname(src))
-        if key == ".":
-            key = basename(realpath(self.export_dir))
+        rel_path = os.path.relpath(src, self.project_path)
```

Comment:
This will not work on the online environment. Please use `relpath(src, self.resources.file_basepath[src])`